### DISCUSSION TITLE
Not a issue but a question On the dec keypads.

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [taligentx/dscKeybusInterface#136] [dscKeybusInterface Discussion]
Hello I was curious if anyone interface a disc keypad with the esp??

**Comment by taligentx** [comment]
Hi @Therealjosephchrzempiec - late response, you can find a partial implementation for the Raspberry Pi by gkovacs81 - this may be a good starting point to implement an ESP8266/ESP32 as a DSC keypad host: https://www.avrfreaks.net/comment/2823396#comment-2823396
